config/sections/mixins/headline.php

```diff
@@ -1,18 +1,25 @@
 <?php
 
+use Kirby\Cms\Helpers;
 use Kirby\Toolkit\I18n;
 
 return [
     'props' => [
         /**
          * The headline for the section. This can be a simple string or a template with additional info from the parent page.
-         * @todo deprecate in 3.7
+         * @todo remove in 3.9.0
          */
         'headline' => function ($headline = null) {
+            if ($headline !== null) {
+                Helpers::deprecated('`headline` prop for sections has been deprecated and will be removed in Kirby 3.9.0. Use `label` instead.');
```

Comment:
@texnixe Since we just only made the changes to the kits and docs. Maybe wait with the warning until 3.8.0`? What do you think?

Reply:
Our sandbox actually is still using it due to the recent additions to the kits only. So really better to wait with the deprecation notice until 3.8.0.